src/components/BookItem.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const BookItem = () => (
+  <div className="bookItem">
+    <span>Book name</span>
+  </div>
```

Comment:
- [ ] Let's add a _Remove_ button to the individual Book. It should not have any functionality yet.

> Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)

Reply:
This is done